storage/database/src/main/java/de/danoeh/antennapod/storage/database/mapper/FeedItemSortQuery.java

```diff
@@ -4,26 +4,29 @@
 import de.danoeh.antennapod.storage.database.PodDBAdapter;
 
 public class FeedItemSortQuery {
+    public static final String ASCENDING = "ASC";
+    public static final String DESCENDING = "DESC";
+
     public static String generateFrom(SortOrder sortOrder) {
-        String sortQuery = "";
+        String sortQuery;
         switch (sortOrder) {
             case EPISODE_TITLE_A_Z:
-                sortQuery = PodDBAdapter.TABLE_NAME_FEED_ITEMS + "." + PodDBAdapter.KEY_TITLE + " " + "ASC";
```

Comment:
I don't think these changes are needed. They just make the PR larger and possibly cause conflicts with other PRs